### DISCUSSION TITLE
[Support]: Can't get IP from supervisor for go2rtc

### DISCUSSION THREAD

**Discussion by AlexxIT** [blakeblackshear/frigate#11333] [General Support] [answered]
### Describe the problem you are having

I think problem because I'm using Supervised installation method.

I ran these commands from Frigate container:
```
root@ccab4aaf-frigate:/opt/frigate# curl -fsSL -H "Authorization: Bearer ${SUPERVISOR_TOKEN}" http://supervisor/network/interface/default/info
curl: (22) The requested URL returned error: 400 Bad Request

root@ccab4aaf-frigate:/opt/frigate# curl -H "Authorization: Bearer ${SUPERVISOR_TOKEN}" http://supervisor/network/interface/default/info
{"result":"error","message":"Interface default does not exist"}
```

### Version

0.13.2-6476f8a

### Frigate config file

```yaml
-
```

### Relevant log output

```shell
2024-05-10 15:33:49.340535621  [INFO] Preparing new go2rtc config...
2024-05-10 15:33:49.383113345  [WARN] Failed to get IP address from supervisor
2024-05-10 15:33:49.629563898  [WARN] Using go2rtc binary from '/config/go2rtc' instead of the embedded one
2024-05-10 15:33:49.629568179  [INFO] Starting go2rtc...
2024-05-10 15:33:49.713558156  18:33:49.713 INF go2rtc platform=linux/amd64 version=1.9.1
2024-05-10 15:33:49.713562120  18:33:49.713 INF config path=/dev/shm/go2rtc.yaml
2024-05-10 15:33:49.714321456  18:33:49.713 INF [api] listen addr=:1984
2024-05-10 15:33:49.714566067  18:33:49.714 INF [rtsp] listen addr=:8554
2024-05-10 15:33:49.714674495  18:33:49.714 INF [webrtc] listen addr=:8555/tcp
```

### FFprobe output from your camera

```shell
-
```

### Operating system

Debian

### Install method

HassOS Addon

### Coral version

CPU (no coral)

### Network connection

Wired

### Camera make and model

-

**Comment by NickM-27** [comment]
@felipecrs may know more than me about what is going on here, I am unsure why this would fail if it works in HA OS

**Reply by AlexxIT** [reply]
This is not HA OS. This is Supervised over Ubuntu.

PS. Maybe problem because I'm using Ubuntu. It's not recommended by core developers

**Reply by NickM-27** [reply]
right, I am saying this is known working in HA OS

but looks like we use a different URL https://github.com/blakeblackshear/frigate/blob/386ffbf5a6b02c6b900be84fb87b3090fcd4cb57/docker/main/rootfs/etc/s6-overlay/s6-rc.d/go2rtc/run#L30-L36

**Reply by felipecrs** [reply, marked as answer]
@AlexxIT I believe you should question the support of the Supervised installation method (maybe supervisor repo?) why this API call doesn't work in Supervised installs.

```
curl -H "Authorization: Bearer ${SUPERVISOR_TOKEN}" http://supervisor/network/interface/default/info
```

Maybe it's by design, or maybe it's just a bug. Maybe it's because of Ubuntu as well.

Anyway, that's exactly why I made this only a warning. I knew it would only work under specific conditions: it's a time saver when it works, but doesn't break anything when it doesn't.

**Reply by felipecrs** [reply]
@NickM-27, it's the same URL:

https://github.com/blakeblackshear/frigate/blob/386ffbf5a6b02c6b900be84fb87b3090fcd4cb57/docker/main/rootfs/etc/s6-overlay/s6-rc.d/go2rtc/run#L13-L19
